packages/template-retail-react-app/app/components/drawer-menu/index.jsx

```diff
@@ -97,6 +99,14 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
     const supportedLocaleIds = l10n?.supportedLocales.map((locale) => locale.id)
     const showLocaleSelector = supportedLocaleIds?.length > 1
 
+    useEffect(() => {
+        var spinner = document.getElementById('spinner')
+        var catNav = document.getElementById('category-nav')
+        if (!spinner && catNav) {
+            catNav.setAttribute('aria-busy', 'false')
```

Comment:
Here it seems `aria-busy` is a dynamic value. We can make this more Reacty by using a react state.

1. create a component state for `DrawerMenu`
```jsx
const [ariaBusy, setAriaBusy] = useState("true")
```
2. call `setAriaBusy` instead of `setAttribute`
3. Let react handle DOM changes

```jsx
return (<Drawer>
                     <div
                            id="category-nav"
                            aria-live="polite"
                            aria-busy={ariaBusy}
                            aria-atomic="true"
                        >
                        ...
                        </div>
    </Drawer>)
```

Reply:
This definitely seems like a better alternative to accessing the DOM. Thanks for the comments Kevin!